autoprotocol/builders.py

```diff
@@ -2015,14 +2023,21 @@ def incubate_params(
         }
 
 
+class EvaporateModeParamsMode(enum.Enum):
+    rotary = enum.auto()
+    centrifugal = enum.auto()
+    vortex = enum.auto()
+    blowdown = enum.auto()
+
+
 class EvaporateBuilders(InstructionBuilders):
     """
     Helpers for building Evaporate instructions
     """
 
     def __init__(self):
         super(EvaporateBuilders, self).__init__()
-        self.valid_modes = ["rotate", "centrifuge", "vortex", "blowdown"]
+        self.valid_modes = ["rotary", "centrifugal", "vortex", "blowdown"]
```

Comment:
# TODO: Revisit this and find which valid_modes are true

Reply:
I believe the valid modes are rotate and centrifuge, here is the [commit](https://github.com/strateos/autoprotocol-scala/blob/ce7db79a4a80ee686f301932e805d8d9a80af997/src/autoprotocol/src/main/scala/org/autoprotocol/instructions/Evaporate.scala) that I found that makes me think this.